addon-test-support/-private/execution_context.js

```diff
@@ -24,56 +24,22 @@ export function getExecutionContext(pageObjectNode) {
   // first, and only if that hasn't happened, check to see if we're in an
   // RFC232/RFC268 test, and if not, fall back on assuming a pre-RFC268
   // acceptance test, which is the only remaining supported scenario.
-  let integrationTestContext = getIntegrationTestContext(pageObjectNode);
+  let context = getTestContext() || getRoot(node).context;
+
+  const isModuleForIntegration = typeof context === 'object' && context !== null
+    && typeof context.$ === 'function'
+    && !context.element;
```

Comment:
`context.element` exists in the `setup*Test`'s context. When the `element` is not present it's a `moduleFor*`